core/shared/src/main/scala/org/http4s/Charset.scala

```diff
@@ -36,7 +36,7 @@ import java.util.Locale
 import scala.jdk.CollectionConverters._
 
 // scalafix:off Http4sGeneralLinters; bincompat until 1.0
-final case class Charset private (nioCharset: NioCharset) extends Renderable {
+final case class Charset private[http4s] (nioCharset: NioCharset) extends Renderable {
```

Comment:
```
core/shared/src/main/scala/org/http4s/Charset.scala:38:1: error: [Http4sGeneralLinters.nonValidatingCopyConstructor] Case classes with private constructors should be abstract to prevent exposing a non-validating copy constructor
[error] final case class Charset private[http4s] (nioCharset: NioCharset) extends Renderable {
[error] ^
[error] (scalafixAll) scalafix.sbt.ScalafixFailed: LinterError
```